ur_moveit_config/launch/ur_moveit.launch.py

```diff
@@ -312,6 +316,13 @@ def generate_launch_description():
             description="URDF/XACRO description file with the robot.",
         )
     )
+    declared_arguments.append(
+        DeclareLaunchArgument(
+            "publish_robot_description_semantic",
+            default_value="False",
```

Comment:
```suggestion
            default_value="True",
```

In #1036 (targeting rolling) we added this feature enabled by default. I'd suggest to make it the same here, unless you had a specific reason to default to `False`?